src/containers/Header/Header.js

```diff
@@ -6,12 +6,34 @@ import ThemeSwitch from '../../components/ThemeSwitch';
 
 import './Header.css';
 
-const Header = () => (
-  <div className="header">
-    <Sidebar />
-    <LocaleToggle />
-    <ThemeSwitch />
-  </div>
-);
+class Header extends React.Component {
+  state = {
+    showSidebar: false,
+  };
+
+  sidebarClosedHandler = () => {
+    this.setState({ showSidebar: false });
+  };
+
+  sidebarToggleHandler = () => {
+    this.setState(prevState => {
+      return { showSidebar: !prevState.showSidebar };
+    });
+  };
+
+  render() {
+    return (
+      <div className="header">
+        <Sidebar
+          opened={this.state.showSidebar}
+          close={this.sidebarClosedHandler}
+        />
+        <LocaleToggle />
+        <ThemeSwitch />
+        <div onClick={this.sidebarToggleHandler}>HAMBURGER ICO</div>
```

Comment:
sidebarToggleHandler is a bad name for the event handler.
toggleSidebar is better.